addons/sms/models/mail_thread.py

```diff
@@ -33,7 +33,7 @@ def _compute_message_has_sms_error(self):
 
     @api.model
     def _search_message_has_sms_error(self, operator, operand):
-        return ['&', ('message_ids.has_sms_error', operator, operand), ('message_ids.author_id', '=', self.env.user.partner_id.id)]
+        return ['&', ('message_ids.has_sms_error', operator, operand), ('message_ids.author_id.id', '=', self.env.user.partner_id.id)]
```

Comment:
You normally don't have to put .id in a domain about m2o. What was the issue ?

Reply:
Yup, it was a wip that I forgot to remove, I'll delete this from the PR